api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -98,27 +95,6 @@ public Profile getProfile(User user, boolean checkEmailVerification) {
     profile.setInstitutionalAffiliations(user.getInstitutionalAffiliations()
         .stream().map(TO_CLIENT_INSTITUTIONAL_AFFILIATION)
         .collect(Collectors.toList()));
-    EmailVerificationStatus userEmailVerificationStatus = user.getEmailVerificationStatus();
-    // if verification is pending or unverified, need to query MailChimp and update DB accordingly
-    // Note: This is pretty slow, we shouldn't be calling it on every Profile lookup. Leave it
-    // for now as we're in the process of switching off Mailchimp (related: RW-705).
-    if (checkEmailVerification &&
-        !userEmailVerificationStatus.equals(EmailVerificationStatus.SUBSCRIBED)) {
-      if (userEmailVerificationStatus.equals(EmailVerificationStatus.UNVERIFIED) && user.getContactEmail() != null) {
-        try {
-          mailChimpService.addUserContactEmail(user.getContactEmail());
```

Comment:
`mailChimpService` I believe is now unused by this service

Reply:
I have a large-ish PR for mailchimp removal coming and was planning on doing that there. Can do it in this one though.

Reply:
Done.